src/hooks/useRelatedVideos.js

```diff
@@ -23,11 +23,11 @@ const useRelatedVideos = (query) => {
       console.log(error);
       setIsLoading(false);
     }    
-  }
+  }, []);
```

Comment:
If your useCallback doesn't have any dependency it won't do much as it will be created on each render. Perhaps use a regular function instead.

Reply:
I used useCallback so that I can use the function as a dependency on `useEffect`